src/karts/skidding.cpp

```diff
@@ -443,6 +443,9 @@ void Skidding::update(float dt, bool is_on_ground,
                                              bonus_speed, bonus_speed,
                                              bonus_force, bonus_time,
                                              /*fade-out-time*/ 1.0f);
+                    // Play custom character sound (weee!)
+                    m_kart->playCustomSFX(SFXManager::CUSTOM_SKID);
+                    m_kart->m_controller->handleZipper(true);                   
```

Comment:
There's multiple ways to do this. One way could be to have a separate method showZipperVisualEffects() that would be called both by handleZipper and your new method, for instance, that would handle the graphical effects only. There are probably other ways, this is just an idea